tests/integration/security/testdata/authz/v1beta1-custom.yaml.tmpl

```diff
@@ -111,3 +111,23 @@ spec:
     protocol: GRPC
   resolution: STATIC
 ---
+
+# The following policy applies the CUSTOM action with the ext-authz-tcp provider on workload from service account a for path /custom.
+
+apiVersion: "security.istio.io/v1beta1"
+kind: AuthorizationPolicy
+metadata:
+  name: policy-a-b-tcp
+  namespace: "{{ .Namespace }}"
+spec:
+  selector:
+    matchLabels:
+      "app": "b"
```

Comment:
do not apply to "b" again, we already have a policy for b. Apply it to a new workload, like "f", or you can reuse and modify the policy for "e" as it's already using gRPC ext-authz.